lms/djangoapps/static_template_view/tests/test_views.py

```diff
@@ -5,14 +5,25 @@
 from django.core.urlresolvers import reverse
 from django.test import TestCase
 
+from openedx.core.djangoapps.site_configuration.tests.test_util import with_site_configuration_context
+
 
 class MarketingSiteViewTests(TestCase):
     """ Tests for the marketing site views """
 
+    def setUp(self):
+        """
+        Sets up the test.
+        """
+        from django.test.utils import setup_test_environment
+        setup_test_environment()
```

Comment:
According to the [documentation](
https://github.com/django/django/blob/e3cf8d2e94fe5f0b6d41774edbab0a9800b397ff/django/test/utils.py#L99-L105) of this function, this is for global test setup and should be run once by the test runner, and not for individual tests.  What's the idea of calling it here?

Reply:
When running the tests locally last week it said it needed it. Deleting it and running it now, it doesn't. Deleted the method.